packages/material-ui/src/TabScrollButton/TabScrollButton.js

```diff
@@ -44,7 +44,7 @@ const TabScrollButtonRoot = experimentalStyled(ButtonBase, {
     width: '100%',
     height: 40,
     '& svg': {
-      transform: 'rotate(90deg)',
+      transform: `rotate(${styleProps.isRtl ? -90 : 90}deg)`,
```

Comment:
To fix this one.
![image](https://user-images.githubusercontent.com/18292247/119969974-6cda8200-bfd9-11eb-941f-96fcfb9f0c68.png)

Reply:
Oh nice, it's a regression from v5. We could have made a standalone PR for it. It was working before because of https://github.com/kentcdodds/rtl-css-js/blob/e9bba97b28fff3b62dc22e69e40d26690e757d11/src/__tests__/index.js#L279.

Reply:
split the PR to https://github.com/mui-org/material-ui/pull/26527

Reply:
Perfect